### ISSUE TITLE
Synopsysctl bad Error when creating in namespace that already exists

### ISSUE THREAD

**Issue by mphammer** [blackducksoftware/synopsys-operator#452]
**What happened**:
When you attempt to create a resource into a namespace that already exists there is an error message. This is confusing to the user because it seems like the resource failed to create (however, it just failed to create the namespace). 

**What you expected to happen**:
No error messages if the resource is successfully created. 

**How to reproduce it (as minimally and precisely as possible)**:
- kubectl create ns bd
- synopsysctl create blackduck bd

**Screenshots/additional context**:
```
INFO[0000] creating Black Duck bd instance...           
INFO[0000] creating Namespace bd                        
ERRO[0000] error in creating the namespace due to Namespace: namespaces "bd" already exists 
```

**Comment by mphammer** [comment]
Being resolved in PR #421

**Comment by rrati** [comment]
Resolved in #421
